From cf8c4cae1e8f991682598cb4cfdf1596a57e33cf Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@users.noreply.github.com>
Date: Mon, 20 Nov 2017 16:25:04 -0600
Subject: [PATCH] [dev/release/2.0] Update BuildTools (#271)

* Update BuildTools to 2.0.0-servicing-02217-01

* Update CoreFX to unblock BuildTools upgrades

* Update Standard to unblock BuildTools upgrades
---
 BuildToolsVersion.txt | 2 +-
 src/corefx            | 2 +-
 src/standard          | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/BuildToolsVersion.txt b/BuildToolsVersion.txt
index 96f2cb5..5e40414 100644
--- a/BuildToolsVersion.txt
+++ b/BuildToolsVersion.txt
@@ -1 +1 @@
-2.0.0-servicing-02120-02
+2.0.0-servicing-02217-01
diff --git a/src/corefx b/src/corefx
index 4de338b..0912774 160000
--- a/src/corefx
+++ b/src/corefx
@@ -1 +1 @@
-Subproject commit 4de338bfa8d84c070247ea0903abb873f7c9a5c4
+Subproject commit 091277427f67e2f9fc25c78d5640ea3a930ace72
diff --git a/src/standard b/src/standard
index 557d9e7..f521a0e 160000
--- a/src/standard
+++ b/src/standard
@@ -1 +1 @@
-Subproject commit 557d9e78bc399abbd5a950e7719c14cf5d79b7f9
+Subproject commit f521a0ed6bcf6326bee829bc478c9e6529c2b0e2
-- 
2.7.4

